pkg/controller/node/rate_limited_queue.go

```diff
@@ -201,10 +203,11 @@ func (q *RateLimitedTimedQueue) Try(fn ActionFunc) {
 
 // Adds value to the queue to be processed. Won't add the same value a second time if it was already
 // added and not removed.
-func (q *RateLimitedTimedQueue) Add(value string) bool {
+func (q *RateLimitedTimedQueue) Add(value string, uid interface{}) bool {
```

Comment:
Comment above might need clarification whether "same value" is by value or by uid.
I see UniqueQueue does `if q.set.Has(value.Value) { return false }` where `.set` is a set of strings, so by value.
But then it's unclear what exactly "anything that can identify the value" means.
Also consider tests with non-1:1 correspondence of values and uids.